ssas/service/main/main.go

```diff
@@ -90,6 +90,16 @@ func init() {
 
 	// used by both `doResetSecret` and `doGetXdata`
 	flag.StringVar(&clientID, "client-id", "", "a system's client id")
+
+	appName := os.Getenv("NEW_RELIC_APP_NAME")
```

Comment:
I'm surprised this wasn't already in place for ACO API.  What are the implications to ACO API of adding this?

Reply:
If the agent doesn't get initialized then the logger will warn about it but the app should continue merrily along. This is tested by the [build and test github actions](https://github.com/CMSgov/bcda-ssas-app/pull/89/checks?check_run_id=2973071386) which doesn't provide `NEW_RELIC_APP_NAME` or `NEW_RELIC_LICENSE_KEY` as secrets.

Reply:
🎉